src/data-browser/index.js

```diff
@@ -1,39 +1,30 @@
 import React from 'react'
-import { Switch, Route, Redirect } from 'react-router-dom'
+import { Switch, Route } from 'react-router-dom'
 import Home from './Home'
 import Geography from './datasets/Geography.jsx'
 import MapsGraphs from './maps/MapsGraphs.jsx'
 import NotFound from '../common/NotFound'
 import { withAppContext } from '../common/appContextHOC'
+import { redirectIfNoYearProvided } from '../common/redirectIfNoYearProvided'
 
 const DataBrowser = props => {
   const { publicationReleaseYear } = props.config
 
   return (
-    <div className="DataBrowser App">
+    <div className='DataBrowser App'>
```

Comment:
why the quote style change? while this is jsx, think normally html style tag attributes should be double quotes?

Reply:
sounds good; yeah, jsx should transpile it to correct quotes;
for reference, when you inspect an element in chrome, the double quotes isn't indicative of the source as the browser would convert it regardless:
![image](https://user-images.githubusercontent.com/73617864/175408825-7e99787a-2971-466e-b988-0e682f4dd001.png)
becomes
![image](https://user-images.githubusercontent.com/73617864/175408084-a54cbf3d-f8ea-48ba-853e-8dc6b29487c4.png)